internal/core/metadata/rest_addressable_test.go

```diff
@@ -91,14 +93,14 @@ func TestGetAllAddressables(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-
 			rr := httptest.NewRecorder()
 			var loggerMock = logger.NewMockClient()
 			restGetAllAddressables(
 				rr,
 				loggerMock,
 				tt.dbMock,
-				errorconcept.NewErrorHandler(loggerMock))
+				errorconcept.NewErrorHandler(loggerMock),
+				configuration)
```

Comment:
Move your definition of `configuration` into the test loop so each iteration has its own cleanly initialized instance.